### PR TITLE
Gravi aura influence

### PR DESCRIPTION
**Pull request by mrbasado**

### CODE REVIEW

File: src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx [hunk 1]
```diff
@@ -110,4 +112,67 @@ export const vaults: InfluenceVaultConfig[] = [
...
+      },
+    ],
+    rewardFrequenciesModalConfig: {
+      title: 'Reward Frequency',
+      body: [
+        'graviAURA and BADGER rewards from bribe sales are distributed at the completion of each bi-weekly voting round, after bribes for that round have been sold.',
+      ],
+      points: [
+        ['- 75% of bribe sales (after fees): sold for graviAURA'],
+        ['- 25% of bribe sales (after fees): sold for BADGER'],
```
Comment (axejintao) [inline src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx:136]: can we check if this is correct? i think it was all emissions although i didn't check closely. if there are no badger token rewards in tree distro then i think its not from bribes

File: src/mobx/stores/InfluenceVaultStore.ts [hunk 2]
```diff
@@ -82,20 +82,34 @@ class InfluenceVaultStore {
           sett: vault.vaultToken.toLowerCase(),
         },
       });
-      const harvestConvertedSchedules = badgerTreeDistributions.map((e) => ({
-        token: e.token.id.startsWith('0x0x') ? e.token.id.slice(2) : e.token.id,
+      const { settHarvests } = await this.store.sdk.graph.loadSettHarvests({
```
Comment (axejintao) [inline src/mobx/stores/InfluenceVaultStore.ts:85]: can you create a ticket for a follow up - maybe we could a utility method to the sdk for grabbing all harvests / distributions. 

https://github.com/Badger-Finance/badger-api/blob/main/src/accounts/accounts.utils.ts#L45-L80

something like this

File: src/mobx/stores/InfluenceVaultStore.ts [hunk 3]
```diff
@@ -107,7 +121,8 @@ class InfluenceVaultStore {
   async loadSwapPercentage(vault: VaultDTO) {
     const config = getInfluenceVaultConfig(vault.vaultToken);
     const { provider } = this.store.sdk;
-    if (!provider || this.influenceVaults[vault.vaultToken].swapPercentage !== '' || !config) return;
+    if (!provider || this.influenceVaults[vault.vaultToken].swapPercentage !== '' || !config || !config.poolToken)
+      return;
```
Comment (axejintao) [inline src/mobx/stores/InfluenceVaultStore.ts:125]: can we use braces, stylistically not a fan of this

**Review by axejintao — COMMENTED** [review]
few comments

File: src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx (hunk 1 above)
Comment (mrbasado) [reply src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx:136]: it matches the Figma, i'll confirm with ZK and Trit

File: src/mobx/stores/LockedDepositsStore.ts [hunk 4]
```diff
@@ -64,13 +65,24 @@ class LockedDepositsStore {
...
     ]);
+    let totalTokenBalanceStrategy = BigNumber.from('0');
+    let lockedTokenBalanceStrategy = BigNumber.from('0');
+    if (factoryType === 'voteLocked') {
+      [totalTokenBalanceStrategy, lockedTokenBalanceStrategy] = await Promise.all([
+        await voteLockedDepositContract.lockedBalanceOf(vaultAddress),
+        await voteLockedDepositContract.balanceOf(vaultAddress),
+      ]);
+    } else {
+      const result = await auraLockerDepositContract.lockedBalances(vaultAddress);
```
Comment (axejintao) [inline src/mobx/stores/LockedDepositsStore.ts:82]:
```
        returns (
            uint256 total,
            uint256 unlockable,
            uint256 locked,
            LockedBalance[] memory lockData
        )
        ```

File: src/mobx/stores/InfluenceVaultStore.ts (hunk 2 above)
Comment (mrbasado) [reply src/mobx/stores/InfluenceVaultStore.ts:85]: yeah ill put that and other refactor parts in a new ticket 👍

**Review by axejintao — APPROVED** [review]

**Pull request merged**